spinalcordtoolbox/utils/fs.py

```diff
@@ -13,14 +13,12 @@
 logger = logging.getLogger(__name__)
 
 
-def tmp_create(basename=None):
+def tmp_create(basename):
     """Create temporary folder and return its path
     """
-    prefix = "sct-%s-" % datetime.datetime.now().strftime("%Y%m%d%H%M%S.%f")
-    if basename:
-        prefix += "%s-" % basename
+    prefix = f"sct_{datetime.datetime.now().strftime('%Y-%m-%d_%H:%M:%S')}_{basename}_"
```

Comment:
Oops! I think I've caused an error on native Windows machines by [using the `:` character](https://stackoverflow.com/a/31976060) in a directory name.

```
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\sct_2022-12-23_20:36:28_install-data_qdfid74d'
```

I'll swap this out for `-`.